scripts/ci/manage-operator.sh

```diff
@@ -174,7 +196,7 @@ metadata:
   name: ${SUBSCRIPTION_NAME}
   namespace: ${NAMESPACE}
 spec:
-  channel: alpha
+  channel: ${CHANNEL}
```

Comment:
Where is this set?

Reply:
a few lines above https://github.com/codeready-toolchain/toolchain-cicd/blob/0b9fb79498271fc2bee4d5020b540fa397b6c351/scripts/ci/manage-operator.sh#L144-L152